corehq/ex-submodules/dimagi/utils/management/commands/prime_views.py

```diff
@@ -1,14 +1,18 @@
 from couchdbkit.exceptions import ResourceNotFound
-# http://www.gevent.org/gevent.monkey.html#module-gevent.monkey
-from gevent import monkey; monkey.patch_all()
-import sys
-import gevent
-from gevent.pool import Pool
-from django.core.management.base import BaseCommand
 
-from django.conf import settings
+# https://www.gevent.org/api/gevent.monkey.html#use-as-a-module
+from gevent import monkey
+monkey.patch_all()
```

Comment:
Maybe out of scope for this PR, but this gevent monkey patch can be removed because of

https://github.com/dimagi/commcare-hq/blob/1d8a8e1d24dd1e9600d06a0178475ef385008155/manage.py#L24

Reply:
Nice. I noticed that too. I was keen to keep the actual code changes to a minimum. I'll drop this in a follow-up PR.